includes/wp-admin/class-advanced-settings-fields.php

```diff
@@ -241,14 +241,14 @@ public static function render_following_ui_field() {
 		<p>
 			<label>
 				<input type="checkbox" id="activitypub_following_ui" name="activitypub_following_ui" value="1" <?php checked( '1', $value ); ?> />
-				Display the "Following" interface in the admin menus and settings.
+				<?php \esc_html_e( 'Display the "Following" interface in the admin menus and settings.', 'activitypub' ); ?>
```

Comment:
The strings in advanced settings are purposefully not translated, as this is more of a power-user tool and has a lot of long and complicated strings that we don't want to burden translators with. Please revert the changes in this file.